openmdao/surrogate_models/kriging.py

```diff
@@ -131,7 +131,7 @@ def train(self, x, y):
 
```

Comment:
Revert all of these changes in kriging.py.  The use of param here has nothing to do with openmdao inputs. It's the kriging model hyperparameters.